packages/core-modules/contracts/modules/CoreElectionModule.sol

```diff
@@ -133,37 +139,46 @@ contract CoreElectionModule is IElectionModule, ElectionStorage, OwnableMixin {
         _electionStore().nextNominationPeriodPercent = percent;
     }
 
-    function elect(address[] memory candidates) external view override {
-        uint seatCount = _electionStore().nextSeatCount;
+    function elect(address[] memory candidates) external override {
+        ElectionStore storage store = _electionStore();
+        ElectionData storage electionData = store.electionData;
+
+        uint seatCount = store.nextSeatCount;
 
-        if (candidates.length != seatCount) {
+        if (candidates.length == 0 || candidates.length > seatCount) {
```

Comment:
We don't need to limit it to the seats count. We can include (if needed) a max voting length as another parameter, but we shouldn't limit it here

Reply:
But, if we remove the max amount of candidates to vote, I think we should implement also the distribution of the voting power to all the candidates (equally, at the begining).

e.g., if a whale votes the "first" 100 candidates, any address with less voting power than that will almost be left out, because it would get the minimum bar for affecting the votes higher.

Reply:
As we just talk on DM, adding hard limit by amount of nominees.

Reply:
Didn't add a check for nominees count, just started checking that the candidate being voted is actually a nominee, and with that and the repeated addresses check, we have a hard limit.